webapp/components/ContentMenu/ContentMenu.vue

```diff
@@ -161,14 +161,31 @@ export default {
               callback: () => {
                 this.$emit('unmute', this.resource)
               },
-              icon: 'user-plus',
+              icon: 'eye',
             })
           } else {
             routes.push({
               label: this.$t(`settings.muted-users.mute`),
               callback: () => {
                 this.$emit('mute', this.resource)
               },
+              icon: 'eye-slash',
+            })
+          }
+          if (this.resource.blocked) {
+            routes.push({
+              label: this.$t(`settings.blocked-users.unblock`),
+              callback: () => {
+                this.$emit('unblock', this.resource)
+              },
+              icon: 'user-plus',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:31:02Z" title="Thursday, January 23rd 2020, 6:31:02 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -172,6 +172,23 @@ export default {
               icon: 'user-times',
             })
           }
+          if (this.resource.isBlocked) {
+            routes.push({
+              label: this.$t(`settings.blocked-users.unblock`),
+              callback: () => {
+                this.$emit('unblock', this.resource)
+              },
+              icon: 'user-plus',
```


questionable if we want to use the same icons as the `mute/unmute` or if we should have similar, but slightly different ones

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T15:56:04Z" title="Wednesday, January 29th 2020, 4:56:04 pm +01:00">Jan 29, 2020</time>_

---


definitely different ones when it has different effects! just not sure which one would fit... 🤷‍♀